src/components/street-geo.js

```diff
@@ -146,10 +156,15 @@ AFRAME.registerComponent('street-geo', {
   },
   google3dUpdate: function () {
     const data = this.data;
+    // if data.ellipsoidalHeight, use it, otherwise use data.elevation less constant (deprecated)
+    const height = data.ellipsoidalHeight
+      ? data.ellipsoidalHeight
+      : data.elevation - this.elevationHeightConstant;
+
```

Comment:
You can set this.height in init so you don't have the calculation logic twice in this component.

Reply:
this.height may change, it's used in `google3dCreate` and `google3dUpdate`, so the only other option I could see was to create a small function wrapper but if it's only 2 times then maybe not worth it